generator/test/good_test_data/all_types.expected

```diff
@@ -76,7 +76,7 @@ class Bar extends _Bar with RealmEntity, RealmObject {
   @override
   String get id => RealmObject.get<String>(this, 'id') as String;
   @override
-  set id(String value) => throw RealmUnsupportedSetError();
+  set id(String value) => RealmObject.set(this, 'id', value);
```

Comment:
Why?

Reply:
You can change PKs during migration.

Reply:
This still throws an exception if you're outside of migration.

Reply:
Eventually (when static meta programming lands) these setter should not exist at all, and it will be a compile time error to use them. I would like if we could use `RealmObject.set` (or `instanceSchema.properties['id'].setValue()`) directly where needed.

Reply:
I relent 😄  We will revisit this once static meta programming has landed, and we re-work the generator.